### PR TITLE
Updated Gutenberg Blocks

### PR DESCRIPTION
**Pull request by vrundakansara**
### Description
`composer update` command shall now update Gutenberg Templates.

P.S. This will require your one-time setup for an access token from [your profile](https://github.com/settings/tokens/new). You can set it from here. and config by running the below command:
`composer config github-oauth.github.com YOUR_TOKEN`

### CODE REVIEW

**Review by AbhijitNage123 — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -15,12 +15,23 @@
...
+		  "type": "vcs",
+		  "url": "git@github.com:brainstormforce/gutenberg-templates.git"
+		}
+	],
   "extra": {
     "installer-paths": {
         "lib/{$name}/": [
             "brainstormforce/astra-notices"
+        ],
+        "lib/ast-block-templates/": [
```
Comment (Nikschavan) [inline composer.json:33]: Why not keep this same as the repo name? This makes it easier to understand where this is coming from easily

File: composer.json [hunk 2]
```diff
@@ -15,12 +15,23 @@
     "test": "vendor/bin/phpunit"
   },
   "require": {
-    "brainstormforce/astra-notices": "^1.1"
+    "brainstormforce/astra-notices": "^1.1",
+    "brainstormforce/gutenberg-templates": "^1"
```
Comment (sandeshjangam) [inline composer.json:19]: @vrundakansara  PHPCS is failing in actions. Seems like private repo. @Nikschavan has done something like a token for another repo.

Failed to download brainstormforce/gutenberg-templates from dist: The "https://api.github.com/repos/brainstormforce/gutenberg-templates/zipball/ee1aa71d554e8e41b8b8bd85b291c379fc4db116" file could not be downloaded (HTTP/2 404 )

Comment (vrundakansara) [reply composer.json:19]: Yes I am checking the same @sandeshjangam

**Review by RajkiranBagal — APPROVED** [review]

File: composer.json (hunk 2 above)
Comment (vrundakansara) [reply composer.json:19]: It has been fixed now @sandeshjangam

**Review by supriyajjadhav — APPROVED** [review]

**Review by Nikschavan — APPROVED** [review]

**Review by sandeshjangam — APPROVED** [review]

**Pull request merged**